web/src/title.js

```diff
@@ -0,0 +1,14 @@
+import Vue from 'vue';
+
+const setPageTitle = (el, binding) => {
+  if (binding && binding.value) {
+    document.title = `${binding.value} - DANDI Archive`;
+  } else {
+    document.title = 'DANDI Archive';
+  }
+};
+
+Vue.directive('page-title', {
+  inserted: setPageTitle,
```

Comment:
I initially just copied that snippet from an example in some GH issue, but after adding the `unbind` to the directive, I think `inserted` works better.

Normally, the events go `unbind`->`inserted`->`updated`. When I switched `inserted` to `bind`, the `bind` would sometimes fire before the previous directive unbound (`bind`->`unbind`->`updated`), so until the `updated` event fired there would be a split second where the title would render as `DANDI Archive` rather than the correct title. I have yet to see this happen using `inserted` so I'm sticking with that.

I will add a comment though.

Reply:
Interesting, thanks for the info.